src/cmd/clone.cpp

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "clone.h"
+
+#include <ddb.h>
+
+#include <fstream>
+#include <iostream>
+
+#include "dbops.h"
+#include "exceptions.h"
+
+namespace cmd {
+void Clone::setOptions(cxxopts::Options& opts) {
+    // clang-format off
+        opts
+			.positional_help("[args]")
+			.custom_help("clone (tag|url)")
+			.add_options()
+			("t,target", "Repository tag or full url", cxxopts::value<std::string>())
+			("f,folder", "Target folder", cxxopts::value<std::string>());
```

Comment:
This should be optional: 

```
ddb clone pierotofy/sheffield_park
```
 --> clone into `sheffield_park` directory (if it doesn't exist, error out if directory already exists).

Reply:
Addressed in https://github.com/DroneDB/DroneDB/pull/190/commits/673801c6c48772d8e380c16fda9cf2bcba0724ec